config/settings-env/endpoints.json

```diff
@@ -550,6 +550,20 @@
         "timeout": "600s",
         "input_query_strings": []
       },
+      {
+        "endpoint": "/v1beta/users/{user_id}/pipelines/{pipeline_id}/trigger:stream",
```

Comment:
Does this endpoint actually work? The thins is KrakenD doesn't support colon in the endpoint URL ([ref](https://gophers.slack.com/archives/CANCJD0V8/p1656523770018559?thread_ts=1656462092.785329&cid=CANCJD0V8)).